Add test plans for lecturer interface (#114)
Gamify-IT/docs: dev-manuals/test-plans/lecturer-interface-frontend/plans/u-lecturer-interface-09-edit-area-topic-name.md

# Lecturer-Interface: Edit area topic name in world view (`u.lecturer-interface-9`)

Version: V1.0, 09.08.2022 \
Author: Max Kästner

## Description

> The topic name of a world or dungeon (which is linked to the world of a view) can be changed on the world view.

Reviewer: ```suggestion
The topic name of an area can be changed on the area view.
```

## Precondition

> At least one course is created. The tester is on the view of a world of a course (`/courses/<course-id>/worlds/<world-index>`)

Reviewer: ```suggestion
At least one course with a preconfigured area is created. The tester is on the view of that area.
```

## Postcondition

The topic name of an area was updated and is shown in the view.

## Typical procedure

1. Click on an edit topic name button of an area (select world/dungeon by static name)

> 2. A input field shows up

Reviewer 2: ```suggestion
2. An input field shows up
```
